cmd/aries-js-worker/package.json

```diff
@@ -7,8 +7,9 @@
   "scripts": {
     "build:node": "webpack --config ./webpack.config.node.js --mode=production",
     "build:web": "webpack --config ./webpack.config.web.js --mode=production",
```

Comment:
@llorllale @sudeshrshetty I wonder if `web` and `node` names should include that they are wasm. (followup).